lib/TopStoriesFeed.jsm

```diff
@@ -86,10 +86,14 @@ this.TopStoriesFeed = class TopStoriesFeed {
     }
   }
 
-  uninit() {
+  async clearCache() {
     this.storiesLoaded = false;
```

Comment:
you probably still want to have `.storiesLoaded` be false in `uninit`

Reply:
Agreed, and it also allowed me to find a test I probably should have had but didn't.